### DISCUSSION TITLE
O que vocês estão achando do Svelte?

### DISCUSSION THREAD

**Discussion by wilsonneto-swe** [frontendbr/forum#1937] [Discussão]
Fala galera!

Gostaria de opiniões sobre svelte, alguém ai já usando? Gostando? O que veem no futuro da tecnologia?

Abraços!

**Comment by HelenDias** [comment]
Fala, Wilson!

Não estou usando em prod ainda, mas estamos com um projeto na [Juntos](https://www.juntossomosmais.com.br/) que está usando Svelte + Sapper.
Está sendo uma experiência bem legal, o código em Svelte fica minúsculo e, ao mesmo tempo, legível. O projeto roda bem rápido também, o que pra mim é uma super vantagem hahah
Estamos fazendo quase tudo seguindo a documentação do próprio Svelte (e do Sapper) e um ou outro repositório de exemplo que achamos no Github.
A dificuldade que encontramos é em achar material de apoio em alguns casos de uso. Por exemplo, queríamos adicionar tradução com i18n. Tinha uns exemplos de Svelte + i18n, mas no nosso caso precisávamos de um exemplo com Svelte + Sapper + i18n. Depois de muito tempo procurando, nos deparamos com [esse repositório](https://github.com/kaisermann/sapper-template-i18n) do [@kaisermann](https://github.com/kaisermann), que é uma super referência de Svelte 😉 

No geral gostei bastante da tecnologia, mas não é tão fácil encontrar conteúdo sobre Svelte :)

**Reply by ninetails** [reply]
[Repositório de componentes](https://sveltesociety.dev/components) :D

Postando só pq n aparece no Google Search

**Reply by HelenDias** [reply]
> [Repositório de componentes](https://sveltesociety.dev/components) :D
> 
> Postando só pq n aparece no Google Search

Sensacional, @ninetails! Cheguei a ver algumas coisas no sveltesociety também e achei muito legal ❤️ 
Com o tempo mais conteúdos de Svelte vão surgindo haha
No começo do Vue era bem difícil de encontrar algumas coisas também

**Comment by rafaelfachinelli** [comment]
Será que o Svelte irá se tornar popular nas empresas? São tantas tecnologias e para mim que está começando, focar em um framework pouco requisitado nas vagas não parece uma boa ideia.

**Reply by iagobruno** [reply]
Eu acho difícil o Svelte se tornar popular a curto prazo, então por enquanto pode focar nos outros frameworks mais consolidados no mercado.

**Reply by ninetails** [reply]
O React demorou uns bons anos pra se tornar o "gigante de share de mercado" que é hoje.

Eu pessoalmente gosto muito das idéias do Rich Harris e da idéia em volta da concepção do Svelte. Fui brincar só na versão 3 mas acompanhava desde a versão 2.

Ainda acho que o case mais foda foi da [galera da Stone ter utilizado nos PDV das máquinas](https://github.com/stone-payments/pos-mamba-sdk), ainda que na v2 e temos o grande e fodido @kaisermann ajudando no core ;D E por ajuda da comunidade, agora tb tem TypeScript.

Acho que só falta popularizar mesmo, mas também não estou otimista que será a tão curtíssimo prazo. Ainda acho que isso vai acontecer de forma mais orgânica.

**Reply by kaisermann** [reply]
Não to mais la na Stone, mas o Svelte foi A parada pra rodar web-apps fáceis de desenvolver enquanto mantendo uma performance near-native (nos padrões de uma S920 😆). Volta e meia eu fico pensando "ia ser uma sdk tão bela na v3 do svelte...." hahahahah

**Reply by kaisermann** [reply]
Desde a v3 o Svelte ganhou um BOOM de atenção muito grande. O pessoal do core tá focando em terminar o @svelte/kit que vai afunilar a DX em uma ferramenta oficial. Tem coisa boa e mais facilidades vindo ai

**Reply by ninetails** [reply]
Caraca summonei o boss do jogo 🤣 

Que trouxe mais bagulho pra gente estudar 😭 lol

**Comment by bardobrado** [comment]
Ressuscitando o tópico, dia 24 de Março, vai sair a versão beta do Sveltekit, mas já está liberado para testes. Confesso que estou achando demais.
